setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
 	janus
 	ansible-runner
 	websockets
-	drools_jpy == 0.3.4
+	drools_jpy == 0.3.5
```

Comment:
@tarilabs the drools_jpy 0.3.5 is available now.

Reply:
Awesome!🚀💪